app/templates/components/popup-modal.hbs

```diff
@@ -0,0 +1,26 @@
+<BsModal  @open = {{isPopupOpen}} as |Modal|>
+    <div class= "popup-modal">
+        <Modal.header>{{svg-jar icon width='30px' height="30px" }}</Modal.header>
+        <Modal.body><div class="modal-text">{{modal-title}}</div></Modal.body>
+        <Modal.footer as |footer|>
+
+        {{!--Todo: To reload the router, once the save / delete pop-ups got closed  --}}
+       
+       {{!-- {{#each popup-btns as |btn|}}
+        <BsButton class="popup-buttons" 
+           @onClick={{action (action 'buttonAction' btn.actionName)}}
+        >{{btn.title}}</BsButton>
+       {{/each}}   --}}
+
+        {{#each popup-btns as |btn|}}
+        {{#link-to 'users' class="popup-link" invokeAction=(if (not-eq btn.actionName 'null') (action (action 'buttonAction' btn.actionName))) 
```

Comment:
As this is a common component, Instead of hard coding the route name, we can pass from the parent component.